frontend/packages/kubevirt-plugin/src/utils/redux/id-reference.ts

```diff
@@ -13,7 +13,7 @@ export const makeIDReferences = (resources: FirehoseResourceEnhanced[]): IDRefer
       resource.fieldSelector,
       resource.name,
     );
-    acc[resource.prop] = ['sdkK8s', makeReduxID(resource.model, query)];
+    acc[resource.prop] = ['k8s', makeReduxID(resource.model, query)];
```

Comment:
This is to prevent breaking the kubevirt plugin as we revert back to the `k8s` redux key (discussed at arch sync and will be a smoother transition to keep the same key).

See https://github.com/openshift/console/pull/10681